hack/e2e-test.sh

```diff
@@ -4,15 +4,25 @@ export KIND=$PWD/bin/kind
 
 function cleanup {
     if [ $USE_EXISTING_CLUSTER == 'false' ] 
-    then 
-        $KIND delete cluster --name $KIND_CLUSTER_NAME
+    then
+        if [ ! -d "$ARTIFACTS" ]; then
+            mkdir -p "$ARTIFACTS"
+        fi
+        kubectl logs -n kube-system kube-scheduler-kind-control-plane > $ARTIFACTS/kube-scheduler.log || true
```

Comment:
Perhaps we can do this in a separate PR but since ARTIFACTS is set as the `$PROJECT_DIR/bin` folder [here](https://github.com/kubernetes-sigs/jobset/blob/main/Makefile#L40) think we should think about storing logs in a separate directory, rather than the bin folder.

Reply:
I can access them from the artifacts tab on prow

Reply:
ah ok i see